pkg/metrics/metrics.jsx

```diff
@@ -812,7 +812,7 @@ class CurrentMetrics extends React.Component {
             );
         }
 
-        const rowWrapperIface = row => ({ 'data-interface': row[0] });
+        const rowWrapperIface = row => ({ 'data-interface': row[0].props?.children ?? row[0] });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19339-20230918-213330-17e85d57-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
This is actually correct -- The fallback is never executed. So this can at least be simplified.

Reply:
Done.